### PR TITLE
[878] Switch to a URL based kind for the selection

### PR DESCRIPTION
**Pull request by sbegaudeau**
Bug: https://github.com/eclipse-sirius/sirius-components/issues/878
Signed-off-by: Stéphane Bégaudeau <stephane.begaudeau@obeo.fr>

### CODE REVIEW

**Review by gcoutable — CHANGES_REQUESTED** [review]
Shouldn't [this](https://github.com/eclipse-sirius/sirius-web/blob/master/frontend/src/views/edit-project/EditProjectView.tsx#L135) be `siriusWeb://document` ?

**Comment by sbegaudeau** [comment]
Indeed that seems to be a mistake, I will fix it.

File: frontend/src/views/edit-project/EditProjectView.tsx [hunk 1]
```diff
@@ -132,11 +132,11 @@ export const EditProjectView = () => {
         onRepresentationSelected={onRepresentationSelected}
         readOnly={false}>
         <TreeItemContextMenuContribution
-          canHandle={(item) => item.kind === 'Document'}
+          canHandle={(item) => item.kind === 'siriusWeb://document'}
           component={DocumentTreeItemContextMenuContribution}
         />
         <TreeItemContextMenuContribution
-          canHandle={(item) => item.kind.includes('::')}
+          canHandle={(item) => item.kind.startsWitth('siriusComponents://semantic')}
```
Comment (gcoutable) [inline frontend/src/views/edit-project/EditProjectView.tsx:139]: There is a typo, should be `startsWith`

**Review by gcoutable — CHANGES_REQUESTED** [review]

File: frontend/src/views/edit-project/EditProjectView.tsx (hunk 1 above)
Comment (sbegaudeau) [reply frontend/src/views/edit-project/EditProjectView.tsx]: Fixed

**Review by gcoutable — APPROVED** [review]

**Pull request merged**